legend-sdlc-server/src/main/java/org/finos/legend/sdlc/server/gitlab/auth/GitLabUserContext.java

```diff
@@ -65,46 +65,83 @@ public GitLabApi getGitLabAPI(boolean redirectAllowed)
         {
             GitLabSession gitLabSession = getGitLabSession();
             GitLabToken token = gitLabSession.getGitLabToken();
+            GitLabTokenResponse tokenResponse;
```

Comment:
I don't think this variable needs to be declared up here. From what I can see, it's only used in the try block on lines 79-87.